redbeams/src/main/java/com/sequenceiq/redbeams/service/network/NetworkParameterAdder.java

```diff
@@ -42,6 +42,8 @@
     @VisibleForTesting
     static final String SUBNET_FOR_PRIVATE_ENDPOINT = "subnetForPrivateEndpoint";
 
+    static final String EXISTING_PRIVATE_DNS_ZONE_ID = "existingPrivateDnsZoneId";
```

Comment:
Add `@VisibleForTesting`.

Reply:
I refactored this part: removed all the VisibleForTesting flags and duplicated the constants in the test. Also, added coverage for other providers.